scripts/generateTranslations.js

```
@@ -0,0 +1,95 @@
+    continue;
+  }
+
+  if (line.startsWith('"')) {
+    const chunk = parseQuoted(line);
+    if (activeField === "msgid" && currentId !== null) {
+      currentId += chunk;
+    } else if (activeField === "msgstr") {
+      currentMsgstr[activeIndex] = (currentMsgstr[activeIndex] || "") + chunk;
+    }
+  }
+}
```

ESLint

[error] iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations.

(no-restricted-syntax)